### PR TITLE
adding new faq

### PR DESCRIPTION
**Pull request by mwweinberg**

### CODE REVIEW

File: FAQ.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+This evolving FAQ covers questions related to to the ml5.js [license](https://github.com/ml5js/Code-of-Conduct/blob/main/LICENSE.md) and [Code of Conduct](https://github.com/ml5js/Code-of-Conduct).  Please open an [issue](https://github.com/ml5js/Code-of-Conduct/issues) if you have a question that is not covered by this document.
```
Comment (shiffman) [inline FAQ.md:1]:
```suggestion
# Frequently Asked Questions

This evolving FAQ covers questions related to to the ml5.js [license](https://github.com/ml5js/Code-of-Conduct/blob/main/LICENSE.md) and [Code of Conduct](https://github.com/ml5js/Code-of-Conduct).  Please open an [issue](https://github.com/ml5js/Code-of-Conduct/issues) if you have a question that is not covered by this document.
```

File: FAQ.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+The ml5.js license is a legal agreement that allows users to make use of the ml5.js library. The Code of Conduct has rules you agree to follow when you accept that license.   
+
+The ml5.js license allows users to use the ml5.js library as long as they follow a small number of rules (called “terms”).  One of the terms is that you agree to follow the ml5.js Code of Conduct.  The license makes it a legal obligation to follow the rules set out in the Code of Conduct.  
+
+## Who owns the copyright in ml5.js?
+
+The ml5.js library is made up of thousands of individual contributions from the ml5.js community.  These take the form of commits to the ml5.js repo.  Each contributor owns the copyright in their contribution and licenses the contribution to the rest of the ml5.js community using the same license that governs ml5.js as a whole.  
+
+This means that there is no central ml5.js organization that owns the copyright to the ml5.js library.
```
Comment (shiffman) [inline FAQ.md:13]: Is this something we might eventually change if we were to formalize an entity that maintains ml5.js? For example, if this license were to be applied to p5.js what would this language say regarding The Processing Foundation?

**Review by shiffman — APPROVED** [review]
This looks great! I added one tiny (trivial) change and asked one question (which is not a roadblock to merging). Thank you @mwweinberg I think we can add this now and revise as needed going forward!

File: FAQ.md (hunk 2 above)
Comment (mwweinberg) [reply FAQ.md:15]: Maybe, but even if we formalized we might not.  Basically, in order for a central organization to own the copyright we would probably need to institute a contributor agreement and go back to everyone who had contributed in the past to try and convince them to assign their copyright to the new organization.  There might be reasons to do this, but there are plenty of OSS projects that work fine without it.  As you might imagine, not everyone is a big fan of contributor agreements.

Comment (shiffman) [reply FAQ.md:15]: Ah, I understand better now, thank you for the explanation!

**Pull request merged**